### DISCUSSION TITLE
My searx instance has no CSS

### DISCUSSION THREAD

**Discussion by justsharan** [searx/searx#2520] [Installation issue: custom]
I just finished going through the setup guide for searx and I have it set up with nginx. When I go to the site, it doesn't have any CSS... just plain text. Any reason why?

<img width="1406" alt="Screen Shot 2020-11-07 at 11 55 44 AM" src="https://user-images.githubusercontent.com/53885408/98447082-2cc0ba00-20f0-11eb-9811-245174b25a51.png">

**Comment by unixfox** [comment]
Please paste the logs of your Searx installation and also the logs found in the "network" tab from the dev tools of your web browser.

**Comment by justsharan** [comment]
Never mind, I was able to fix it.

**Reply by edbrenner** [reply]
Hi, having same problem, can you share your solution ? 
Thanks.

**Reply by officereso** [reply]
My solution was to make sure the proxy had the correct permissions to read everything in the searx directory. I added nginx to the searx group and made sure every file in the searx directory was owned by searx and could be both read and executed

**Reply by edbrenner** [reply]
Thanks for reply, I have added nginx user www-data to searx group, restarted both nginx and searx but no change. Strange behavior: it used to work fine after the installation but stopped just after the first post-install server reboot. It is not only about missing css. Once I click on any button on page I got "page not found"... 

- /usr/local/searx is owned by searx : searx
- all files are owned by searx and can be read and executed by searx group member

Any navigation further highly welcomed ! Thank you.

**Reply by officereso** [reply]
You have both /searx (or / if your using a sub domain) and /searx/static being forwarded correctly as specified here https://searx.github.io/searx/admin/installation-nginx.html ?

**Reply by edbrenner** [reply]
I finally fixed it! The problem was in /etc/searx/settingx.yml in server part: 
My base_url was set to http://something.xx/searx/ and once I set it back to default -  **base_url : False** then it started to work. 

So with this one it works:

```
server:
    port : 8888
    bind_address : "127.0.0.1" # address to listen on
    secret_key : "xxxxxxxxxxxxxxxxxxxxxxx" # change this!
    base_url : False                # previously: http://something.xx/searx/ - this was not working
```

@officereso : thanks for navigation! What finally helped was to go via step by step instructions at searx doc page.

**Comment by izyuumi** [comment]
Hello. I am having the same problem right now. Would it be possible for you to let me know what you did to fix it? @SinistreCyborg Thanks!

**Comment by officereso** [comment]
What was your solution

**Comment by justsharan** [comment]
I'm actually not sure what I did to fix it... I tried re-installing it after a couple days and it just worked. I think I might've forgotten a step somewhere. Not sure.

**Reply by return42** [reply]
Maybe you forgot to proxy the static files or missed to give read access to the proxy, see comments from @officereso 

- https://github.com/searx/searx/discussions/2520#discussioncomment-409953
- https://github.com/searx/searx/discussions/2520#discussioncomment-410386
